app/src/views/CookiePolicy/index.tsx

```diff
@@ -0,0 +1,381 @@
+                                                        withUnderline
+                                                        external
+                                                    >
+                                                        {strings.policyProtectionOfPersonalDataLink}
+                                                    </Link>
+                                                ),
+                                            },
+                                        )}
+                                    </div>
+                                    <div>
+                                        {!accepted && resolveToComponent(
+                                            strings.storageSecurityQuestionsAboutDataDescription4,
+                                            {
+                                                termsLink: (
+                                                    <Link
+                                                        href="https://www.ifrc.org/data-protection"
+                                                        withUnderline
+                                                        external
+                                                    >
+                                                        {strings.dataProtectionPageLink}
+                                                    </Link>
+                                                ),
+                                            },
+                                        )}
+                                    </div>
+                                    <div>{strings.storageSecurityQuestionsDataDescription5}</div>
+                                    <div>
+                                        <strong>
+                                            {!accepted && resolveToComponent(
+                                                strings.storageSecurityQuestionsDataGoEnquires,
+                                                {
+                                                    termsLink: (
+                                                        <Link
+                                                            href="mailto:im@ifrc.org"
+                                                            withUnderline
+                                                            external
+                                                        >
+                                                            {strings.ifrcGoEnquiresLink}
+                                                        </Link>
+                                                    ),
+                                                },
+                                            )}
+                                        </strong>
+                                    </div>
+                                    <div>
+                                        <strong>
+                                            {!accepted && resolveToComponent(
```

Comment:
What should be displayed when the terms are accepted? Should we have an empty <strong /> tag?